app/controllers/users/sessions_controller.rb

```diff
@@ -41,8 +41,6 @@ def create
     def destroy
       analytics.logout_initiated(sp_initiated: false, oidc: false)
       irs_attempts_api_tracker.logout_initiated(
-        user_uuid: current_user.uuid,
-        unique_session_id: current_user.unique_session_id,
```

Comment:
I think everyone knows this, but for posterity -- these attributes are going to be baked in universally for all events in a separate PR.